src/JSInterop/Microsoft.JSInterop/src/DotNetDispatcher.cs

```diff
@@ -285,15 +286,5 @@ private static Assembly GetRequiredLoadedAssembly(string assemblyName)
             return loadedAssemblies.FirstOrDefault(a => a.GetName().Name.Equals(assemblyName, StringComparison.Ordinal))
                 ?? throw new ArgumentException($"There is no loaded assembly with the name '{assemblyName}'.");
         }
-
-        private static Exception UnwrapException(Exception ex)
```

Comment:
This logic isn't as complex anymore because I'm not trying to build a one-size-fits-all *unwrapper*.